rasa/runrasa

```diff
@@ -0,0 +1,3 @@
+#!/bin/env bash
```

Comment:
Do we need also this file?

Reply:
I guess it is simpler to type ./runrasa than "rasa run -m models --enable-api --log-file out.log" and if in the future we include an action_server to the rasa module we can include it to run in the bash script.

Reply:
Oh, I mean that there is also the file rasa/runrasa.bash with exactly the same.

Reply:
ah lol thanks